common/djangoapps/student/migrations/0017_auto_20190925_1058.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2019-09-25 17:58
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('student', '0016_coursenrollment_course_on_delete_do_nothing'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='courseenrollment',
+            name='mode',
+            field=models.CharField(default='honor', max_length=100),
```

Comment:
Correctly set this as we want it.

Reply:
It won't work to just specify `COURSE_MODE_DEFAULTS` with slug honor in config?

Reply:
Well, that's where this is coming from...
_Because_ we override `COURSE_MODE_DEFAULTS` to include `slug=honor`, when you run migrations, it determines what the schema should be _based_ _on_ that config value.
So by changing the setting, we need to update the schema.

Reply:
Gotcha, sorry for the dumb question, was sort of hoping we could avoid a forked migration. Anyway could you update the migration filename to something descriptive?

Reply:
@kluo Renamed, pushed, squashed.